src/meta/src/hummock/manager/mod.rs

```diff
@@ -937,6 +950,7 @@ impl HummockManager {
             .unwrap()
             .member_table_ids
             .clone();
+
```

Comment:
nits: not related to this PR, but I just noticed that `member_table_ids` is already retrived in L922 so there is no need to get from `current_version` in L947 and we can just assign it.